rootfs/apply.go

```diff
@@ -96,11 +97,12 @@ func Prepare(snapshots Snaphotter, mounter Mounter, layers []ocispec.Descriptor,
 	)
 
 	for _, layer := range layers {
+		layerDigest := digest.Digest(layer.Digest)
```

Comment:
I've filed https://github.com/opencontainers/image-spec/pull/514 to take care of this.